sound/soc/intel/boards/skl_hda_dsp_generic.c

```diff
@@ -203,48 +48,80 @@ static void skl_set_hda_codec_autosuspend_delay(struct snd_soc_card *card)
 	}
 }
 
+#define IDISP_HDMI_BE_ID	1
+#define HDA_BE_ID		4
+#define DMIC01_BE_ID		6
+#define DMIC16K_BE_ID		7
+#define BT_OFFLOAD_BE_ID	8
+
+#define HDA_LINK_ORDER	SOF_LINK_ORDER(SOF_LINK_IDISP_HDMI,  \
+					SOF_LINK_HDA,        \
```

Comment:
it is interesting that checkpatch is not complaining, but it would look better if the lines are aligned, or follow the other files and use 1 less TAB.

Reply:
fixed